src/graph/GoExecutor.cpp

```diff
@@ -403,6 +421,26 @@ void GoExecutor::onVertexProps(RpcResponse &&rpcResp) {
     UNUSED(rpcResp);
 }
 
+std::vector<std::string> GoExecutor::getEdgeNamesFromResp(RpcResponse &rpcResp) const {
+    std::vector<std::string> names;
+    auto spaceId = ectx()->rctx()->session()->space();
+    for (auto &resp : rpcResp.responses()) {
```

Comment:
Do we need to scan all responses from all hosts?

Reply:
We don't need it here, so I will modify it.